generator/subaru/subaru_forester_2017.dbc

```diff
@@ -0,0 +1,39 @@
+CM_ "IMPORT _subaru_preglobal_2015.dbc"
+
+BO_ 355 ES_DashStatus: 8 XXX
+ SG_ Not_Ready_Startup : 4|2@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_On : 16|1@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_Set_Speed : 24|8@1+ (1,0) [0|255] "" XXX
+ SG_ Counter : 40|3@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_Activated : 54|1@1+ (1,0) [0|1] "" XXX
+
+BO_ 881 Steering_Torque: 8 XXX
+ SG_ Steering_Motor_Flat : 0|10@1+ (32,0) [0|16500] "" XXX
+ SG_ Steer_Torque_Output : 16|11@1- (-32,0) [-16500|16500] "" XXX
+ SG_ LKA_Lockout : 27|1@1+ (1,0) [0|1] "" XXX
+ SG_ Steer_Torque_Sensor : 29|11@1- (-1,0) [-1500|1500] "" XXX
+ SG_ Steering_Angle : 40|16@1- (-0.033,0) [-600|600] "" XXX
+
+CM_ SG_ 320 Off_Throttle_2 "Less sensitive";
```

Comment:
Comment lines should go in the same DBC the signal is defined in

Reply:
Done